zh/option/series/funnel.md

```diff
@@ -86,6 +86,32 @@ option = {
 
 数据排序， 可以取 `'ascending'`，`'descending'`，`'none'`（表示按 data 顺序），或者一个函数（即 `Array.prototype.sort(function (a, b) { ... })`）。
 
+## exitShape(string) = 'none'
+
+<ExampleUIControlEnum options="none,rect" default="none" />
+
+漏斗出口形状, 设为 `'rect'` 来展平顶部。
+
+## dynamicHeight(boolean)
+
+<ExampleUIControlEnum options="true,false" default="false" />
+
+设置为`true`使得漏斗图每个漏斗块的高度和数据集对应数值成正比。
+
+## exitWidth(string)
+
+<ExampleUIControlPercent default="0%" />
+
+当dynamicHeight为`true`时，控制出口宽度占入口宽度多少，设置exitWidth可以使图形更接近漏斗形状，而非尖底的金字塔形状。<br/>
+当dynamicHeight为不为true并且showRate为`true`时，控制出口为一个梯形，即出口块上底边是下底边多少。<br/>
```

Comment:
Add links for these options.
No `<br/>`. Please use an empty line for paragraph breaking.